cfme/tests/cli/test_appliance_update.py

```diff
@@ -98,49 +62,133 @@ def test_update_yum(appliance_preupdate, appliance):
     assert appliance.version == appliance_preupdate.version
 
 
-@pytest.fixture(scope="function")
-def enabled_embedded_appliance(appliance_preupdate):
-    """Takes a preconfigured appliance and enables the embedded ansible role"""
-    appliance_preupdate.enable_embedded_ansible_role()
-    assert appliance_preupdate.is_embedded_ansible_running
-    return appliance_preupdate
-
-
-@pytest.fixture(scope="function")
-def update_embedded_appliance(enabled_embedded_appliance, appliance):
-    with enabled_embedded_appliance:
-        red_hat_updates = RedHatUpdates(
-            service='rhsm',
-            url=conf.cfme_data['redhat_updates']['registration']['rhsm']['url'],
-            username=conf.credentials['rhsm']['username'],
-            password=conf.credentials['rhsm']['password'],
-            set_default_repository=True
-        )
-        red_hat_updates.update_registration(validate=False)
-        red_hat_updates.check_updates()
-        wait_for(
-            func=red_hat_updates.checked_updates,
-            func_args=[appliance.server.name],
-            delay=10,
-            num_sec=100,
-            fail_func=red_hat_updates.refresh
-        )
-        if red_hat_updates.platform_updates_available():
-            red_hat_updates.update_appliances()
-        return enabled_embedded_appliance
+@pytest.mark.ignore_stream("upstream")
+def test_update_webui(appliance_with_providers, appliance, request, old_version):
+    """ Tests updating an appliance with providers, also confirms that the
+        provisioning continues to function correctly after the update has completed"""
+    update_appliance(appliance_with_providers)
+
+    wait_for(do_appliance_versions_match, func_args=(appliance, appliance_with_providers),
```

Comment:
Required: add a message with info what we are waiting for
parameter message='' 
Please add to all wait_for